content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -185,6 +185,13 @@ If you have a continued need for a Russian localization of the Cumulocity IoT UI
 
 #### Implemented
 
+
+##### Changes in Gainsight tracking
+
+In efforts to get more insights from our users, we enable tracking for every user that has agreed or rejected the cookie and policy banner. In case of rejection, the user will be tracked without any personal identifiable information (pii) like e-mail, name or IP address. Not tracked are still users that haven't been shown a cookie/policy banner or users that are using dedicated instances without any Gainsight API key set.
+
+Gainsight additionally brings so called in app engagements. This are pop-ups or similar that try to interact with the end-user and collect feedback. If a custom branding is applied, this engagements are never shown to the end-user even if he is tracked.
```

Comment:
"this engagements are never shown to the end-user even if he is tracked." They're set to false by default (for new users), but user can enable them.